packages/synapse-react-client/mocks/entity/mockProject.ts

```diff
@@ -1,22 +1,26 @@
 import {
   ACCESS_TYPE,
   AnnotationsValueType,
-  Entity,
   EntityBundle,
   EntityHeader,
   EntityJson,
   EntityType,
   ObjectType,
+  Project,
   ProjectHeader,
   RestrictionLevel,
 } from '@sage-bionetworks/synapse-types'
 import { MOCK_USER_ID } from '../user/mock_user_profile'
 import { MockEntityData } from './MockEntityData'
+import { generateProject } from '../faker/generateFakeEntity'
+import { times } from 'lodash-es'
 
-const MOCK_PROJECT_ID = 'syn12345'
+export const mockProjectIds = times(20).map(i => i + 10000)
+
+const MOCK_PROJECT_ID = `syn${mockProjectIds[0]}`
```

Comment:
We already have one mock project with curated data. Give it the first ID in the project ID space.